# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/webkitgtk/no-ruby.diff
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- webkitgtk-2.34.0/Source/cmake/WebKitFeatures.cmake.vanilla	2021-10-01 18:01:01.463496556 +0200
+++ webkitgtk-2.34.0/Source/cmake/WebKitFeatures.cmake	2021-10-01 18:01:28.694497960 +0200
@@ -98,7 +98,7 @@
         message(STATUS "Unified builds are disabled when analyzing sources")
         set(ENABLE_UNIFIED_BUILDS_DEFAULT OFF)
     else ()
-        set(ENABLE_UNIFIED_BUILDS_DEFAULT ON)
+        set(ENABLE_UNIFIED_BUILDS_DEFAULT OFF)
     endif ()
 
     WEBKIT_OPTION_DEFINE(ENABLE_3D_TRANSFORMS "Toggle 3D transforms support" PRIVATE ON)
